scipy/interpolate/tests/test_polyint.py

```diff
@@ -192,6 +196,35 @@ def test_high_derivative(self):
             assert_almost_equal(P.derivative(self.test_xs,i),
                                 np.zeros(len(self.test_xs)))
 
+    def test_ndim_derivatives(self):
+        poly1 = self.true_poly
+        poly2 = np.polynomial.Polynomial([-2, 5, 3, -1])
+        poly3 = np.polynomial.Polynomial([12, -3, 4, -5, 6])
+        ys = np.stack((poly1(self.xs), poly2(self.xs), poly3(self.xs)), axis=-1)
+
+        P = KroghInterpolator(self.xs, ys, axis=0)
+        D = P.derivatives(self.test_xs)
+        for i in range(D.shape[0]):
+            assert_almost_equal(np.stack((poly1.deriv(i)(self.test_xs),
```

Comment:
New tests should use `np.assert_allclose`
```suggestion
            np.assert_allclose(np.stack((poly1.deriv(i)(self.test_xs),
```